ftrack_s3_accessor/s3.py

```diff
@@ -287,3 +287,13 @@ def make_container(self, resource_identifier, recursive=True):
 
         s3_object = self.bucket.Object(resource_identifier)
         s3_object.put(Body='')
+
+    def get_url(self, resource_identifier=None):
+        '''Return url for *resource_identifier*.'''
+        s3_object = self.bucket.Object(resource_identifier)
+        try:
+            location = boto3.client('s3').get_bucket_location(Bucket=self.bucket_name)['LocationConstraint']
+            url = "https://s3-{}.amazonaws.com/{}/{}".format(location, self.bucket_name, s3_object.key)
+        except Exception:
```

Comment:
Can we use a more specific exception? Ideally one from boto3. https://boto3.amazonaws.com/v1/documentation/api/latest/guide/error-handling.html

Reply:
yeah looking into the various exceptions but aiming to use the base one from Boto.
` except botocore.exceptions.BotoCoreError`